modules/net-glb/backend-services.tf

```diff
@@ -208,3 +212,150 @@ resource "google_compute_backend_service" "group" {
     }
   }
 }
+
```

Comment:
I would move the regional resources to a `regional-backend-services.tf` file

Reply:
Should I move all regional resources to `regional-` files? My original idea was to keep the resources in same files where it made sense (eg. the file is short enough for comprehension) given the resource config is almost 1:1.

Reply:
Hmmm, maybe only for "fat files" like backends then?

Reply:
I moved the backends to regional file.